config/pipelines/rbac.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: meteor
+  labels:
+    app: meteor-pipeline
+secrets:
+  - name: auth-secret
```

Comment:
Why does the SA need a custom secret? Pipeline Run consumes the SA name only, there's no need for a custom secret, or did I miss something?

https://github.com/tektoncd/pipeline/blob/main/docs/pipelineruns.md#specifying-custom-serviceaccount-credentials

Reply:
very good point, this was for the bot secret to be added to sa, if we wanted to pull `private repo`